examples/dynamo/eg4/solver_mod.x90

```diff
@@ -33,6 +33,11 @@
 ! ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 ! POSSIBILITY OF SUCH DAMAGE.
 ! -----------------------------------------------------------------------------
+! (c) The copyright relating to this work is owned jointly by the Crown,
+! Met Office and NERC 2014.
+! However, it has been created with the help of the GungHo Consortium,
```

Comment:
Agreed. I have tweaked them slightly so have added the necessary text and re-ordered the license text.